framework/Samplers/AdaptiveMonteCarlo.py

```diff
@@ -216,7 +214,6 @@ def localStillReady(self,ready): #,lastOutput=None
       ready is returned
       lastOutput should be present when the next point should be chosen on previous iteration and convergence checked
       lastOutput it is not considered to be present during the test performed for generating an input batch
```

Comment:
there is no lastOutput. Can you modify the docstring and "re-modify" the one you copied this from?